forge-gui/res/cardsfolder/upcoming/glimpse_the_impossible.txt

```diff
@@ -0,0 +1,12 @@
+Name:Glimpse the Impossible
+ManaCost:2 R
+Types:Sorcery
+A:SP$ Dig | Defined$ You | DigNum$ 3 | ChangeNum$ All | DestinationZone$ Exile | RememberChanged$ True | SubAbility$ DBEffect | SpellDescription$ Exile the top three cards of your library. You may play those cards this turn. At the beginning of the next end step, if any of those cards remain exiled, put them into your graveyard, then create a 0/1 colorless Eldrazi Spawn creature token for each card put into your graveyard this way. Those tokens have "Sacrifice this creature: Add {C}."
+SVar:DBEffect:DB$ Effect | RememberObjects$ RememberedCard | StaticAbilities$ STPlay | SubAbility$ DBDelayTrig | ForgetOnMoved$ Exile
+SVar:STPlay:Mode$ Continuous | MayPlay$ True | EffectZone$ Command | Affected$ Card.IsRemembered | AffectedZone$ Exile | Description$ You may play those cards until end of turn.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:DBDelayTrig:DB$ DelayedTrigger | Mode$ Phase | RememberObjects$ Remembered | ConditionDefined$ Remembered | ConditionZone$ Exile | ConditionPresent$ Card.IsRemembered | Phase$ End Of Turn | Execute$ TrigChangeAll | SubAbility$ DBCleanup | TriggerDescription$ At the beginning of the next end step, if any of those cards remain exiled, put them into your graveyard, then create a 0/1 colorless Eldrazi Spawn creature token for each card put into your graveyard this way. Those tokens have "Sacrifice this creature: Add {C}."
```

Comment:
intervening if not handled

Reply:
Does it needs conditionchecksvar ? I don't really understand how to implement this

Reply:
using conditions too is fine (since once they're false they can't become true again) to not create the delayed trigger at all
however they're not that important since what you really need is normal params for the trigger to check at the end step, just like on e. g. _Bank Job_